common/settings.go

```diff
@@ -169,41 +174,41 @@ func ensureInSlice(elem string, slice []string) []string {
 
 // validAccessTokenAuthn() examines if the admins have configured
 // a valid value for the ACCESS_TOKEN_AUTHN envvar.
-func validAccessTokenAuthn(AccessTokenAuthnEnabledEnv bool, AccessTokenAuthnEnv string) (bool){
+func validAccessTokenAuthn(AccessTokenAuthnEnabledEnv bool, AccessTokenAuthnEnv string) bool {
 	if !AccessTokenAuthnEnabledEnv {
 		return true
 	}
 	if AccessTokenAuthnEnv == "jwt" {
 		return true
 	}
-	if AccessTokenAuthnEnv == "opaque"{
+	if AccessTokenAuthnEnv == "opaque" {
```

Comment:
Please remove formatting only changes.  This makes it much more annoying to merge from upstream